make/Makefile.test

```diff
@@ -13,7 +13,7 @@ endif
 
 .PHONY: test-active
 test-active: prepare-test
-	VM_PID=$$(scripts/run_vm.sh vmpid) go run $(GINKGO) $(GINKGO_ARGS) ./tests/wait-active
+	VM_PID=$$(scripts/run_vm.sh vmpid) COS_PASS=cos go run $(GINKGO) $(GINKGO_ARGS) ./tests/wait-active
```

Comment:
The upgrade test requires the legacy password because it boots with an older disk, so the password needs to be added for `test-active` and `test-upgrade`. It always tries to login by the authorized key first and if not it fallbacks to the password method.

I could not find an easy way at ssh package level to set multiple passwords to try.